libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -47,13 +47,12 @@ export const logErrorFor = async (fn, message, tags) => {
   }
 };
 
-export function addMepHighlight(el, source) {
-  let { manifestId } = source.dataset;
-  if (!manifestId) {
-    const closestManifestId = source?.closest('[data-manifest-id]');
-    if (closestManifestId) manifestId = closestManifestId.dataset.manifestId;
-  }
+export function addMepHighlightAndTargetId(el, source) {
+  let { manifestId, targetManifestId } = source.dataset;
+  manifestId ??= source?.closest('[data-manifest-id]')?.dataset?.manifestId;
```

Comment:
??= is new for me. Never seen it used before =). Maybe consider that a lot of people are like me and also haven't seen it used and may be confused ;) I dunno, up to you

Reply:
@denlight, I take your point about the Nullish coalescing assignment operator (I had to look up the name up again today when Chris suggested it ;)), but it is nice and concise and we've already used in a couple of other places in personalization.js so let's keep it. Thanks.